files/image_config/sudoers/sudoers

```diff
@@ -46,6 +46,8 @@ Cmnd_Alias      READ_ONLY_CMDS = /bin/cat /var/log/syslog*, \
 
 
 Cmnd_Alias      PASSWD_CMDS = /usr/local/bin/config tacacs passkey *, \
+                              /usr/local/bin/config radius passkey *, \
+                              /usr/local/bin/config snmp community add *, \
```

Comment:
it supports add/del/replace, and all the command lines contain community string.

Reply:
Fixed, add del and replace commands to PASSWD_CMDS.